packages/libraries/main/src/Options.ts

```diff
@@ -8,12 +8,43 @@ import {
   RankedDictionaries,
   Matchers,
   Matcher,
+  TimeEstimationValues,
 } from './types'
 import l33tTable from './data/l33tTable'
 import translationKeys from './data/translationKeys'
 import TrieNode from './matcher/dictionary/variants/matching/unmunger/TrieNode'
 import l33tTableToTrieNode from './matcher/dictionary/variants/matching/unmunger/l33tTableToTrieNode'
 
+export const timeEstimationValuesDefaults: TimeEstimationValues = {
+  scoring: {
+    0: 1e3,
+    1: 1e6,
+    2: 1e8,
+    3: 1e10,
+  },
+  attackTime: {
+    onlinePerHour: 100,
+    onlinePerSecond: 10,
+    slowHashing: 1e4,
+    fastHashing: 1e10,
```

Comment:
I think it would be better if these names matched the ones that are returned from `estimateAttackTimes`, for consistency.
Something like this (if you accept my renaming suggestion there):
```suggestion
    onlineThrottling_xPerHour: 100,
    onlineNoThrottling_xPerSecond: 10,
    offlineSlowHashing_xPerSecond: 1e4,
    offlineFastHashing_xPerSecond: 1e10,
```